internal/pkg/gitdirty/gitdirty.go

```diff
@@ -23,6 +23,24 @@ func init() {
 	githubStyleHttpRemoteRegexp = regexp.MustCompile(`http[s]?:\/\/(.*?\..*?)\/(.*)`) // Example: https://git.test/testorg/testrepo.git
 }
 
+// GitInstalled checks if the command-line tool `git` is installed
+func GitInstalled() bool {
+	if _, err := exec.LookPath("git"); err == nil {
+		return true
+	}
+	return false
+}
+
+// GetRepoTopLevel returns the path to the root of the repository that contains pathWithinVcs.
+// Equivalent to git rev-parse --show-toplevel
+func GetRepoTopLevel(log hclog.Logger, pathWithinVcs string) (string, error) {
```

Comment:
Getters comment applies here as well. A more descriptive version might be `RepoTopLevelPath`